bindings/python/iota_sdk/utils.py

```diff
@@ -98,7 +101,7 @@ def compute_alias_id(output_id: OutputId) -> HexStr:
         })
 
     @staticmethod
-    def compute_foundry_id(alias_id: str, serial_number: int, token_scheme_kind: int) -> HexStr:
+    def compute_foundry_id(alias_id: HexStr, serial_number: int, token_scheme_kind: int) -> HexStr:
```

Comment:
Is there a type for AliasId?

Reply:
No, also spoke with Kevin about it and we think it's currently not worth it, since you also need to construct the type then

Reply:
Okay, just thought it might already exist.